### PR TITLE
improvements to splash layout for bootstrap5 theme #4885

### LINKED ISSUE #4885: improvements to splash layout for bootstrap5 theme
## Motivation
This theme was based on some my code and there left stuff that shouldn't be there. 

Splash layout is layout for these pages:
- login.php
- signup.php
- fpw.php
- membersonly.php

* moved layout classes (like container and row) to layout file, removed them from tablerender
* removed not officially supported login page as a standalone page
* added splash templates with clean bootstrap 5 markup
* fixed fpw template (wrapper support added as a separate issue and PR)
* fixed login template - added wrappers

### PR DESCRIPTION
**Pull request by Jimmi08**
CLOSES #4885

### Motivation and Context
to have correct bootstrap 5 templates for splash layout

### Description
- in related issue

### How Has This Been Tested?
in boostrap 5  theme and aeolus theme - I am not aware about more bootstrap5 themes.

### Types of Changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [x ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (a change to man pages or other documentation)

### CODE REVIEW

File: e107_core/templates/bootstrap5/fpw_template.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+$FPW_WRAPPER['form']['FPW_CAPTCHA_IMG']  =
+"<div class='row align-items-center my-2'>
+	<label class='col-md-4'>&nbsp;</label>
+	<div class='col-md-8'>{---}</div>
+</div>";
+
+$FPW_WRAPPER['form']['FPW_CAPTCHA_INPUT']  =
+"<div class='row align-items-center my-2'>
+	<label class='col-md-4 col-form-label' for='code-verify'>" . e107::getSecureImg()->renderLabel() . "</label>
```
Comment (CaMer0n) [inline e107_core/templates/bootstrap5/fpw_template.php:18]: PHP code needs to be removed and replaced with a shortcode.

File: e107_core/templates/bootstrap5/login_template.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_SECIMG_SECIMG']  =
+"<div class='row align-items-center my-2'>
+	<label class='col-md-4'>&nbsp;</label>
+	<div class='col-md-8'>{---}</div>
+</div>";
+
+$LOGIN_WRAPPER['page']['LOGIN_TABLE_SECIMG_TEXTBOC']  =
+"<div class='row align-items-center my-2'>
+	<label class='col-md-4 col-form-label' for='code-verify'>" . e107::getSecureImg()->renderLabel() . "</label>
```
Comment (CaMer0n) [inline e107_core/templates/bootstrap5/login_template.php:50]: ```e107::getSecureImg()->renderLabel()``` needs to be placed into a shortcode

Comment (Jimmi08) [reply e107_core/templates/bootstrap5/login_template.php:50]: Weird. I was sure I used core templates as the starting point and it looks like I didn't.

Comment (Jimmi08) [reply e107_core/templates/bootstrap5/login_template.php:50]: @CaMer0n  This solution is used in core signup template too so I did it the same way. .

Comment (CaMer0n) [reply e107_core/templates/bootstrap5/login_template.php:50]: @Jimmi08 Okay. It's good if we get both cleaned up to use shortcodes.

File: e107_themes/bootstrap5/theme_shortcodes.php [hunk 3]
```diff
@@ -2,15 +2,16 @@
...
  * Released under the terms and conditions of the
  * GNU General Public License (http://www.gnu.org/licenses/gpl.txt)
  *
  * e107 Bootstrap Theme Shortcodes. 
  *
 */
 
-
+use JetBrains\PhpStorm\ArrayShape;
+use JetBrains\PhpStorm\Deprecated;
```
Comment (CaMer0n) [inline e107_themes/bootstrap5/theme_shortcodes.php:15]: I believe these lines can be removed.

**Pull request closed without merging**